wolfcrypt/src/asn.c

```diff
@@ -3738,43 +3738,75 @@ static int GetAlgoV2(int encAlgId, const byte** oid, int *len, int* id,
     return ret;
 }
 
-/* Converts Encrypted PKCS#8 to 'traditional' (i.e. PKCS#8 removed from
```

Comment:
Further down, you'll see that I changed the comment on `TraditionalEnc`. The previous one didn't make sense to me.